paddle/fluid/operators/distributed/communicator.cc

```diff
@@ -633,7 +638,6 @@ std::unordered_set<int64_t> GeoSgdCommunicator::SparseIdsMerge(
   auto origin_var_name = DeltaVarToVar(var_name);
```

Comment:
what's the meaning of `DeltaVarToVar `?

Reply:
> what's the meaning of `DeltaVarToVar `?

change Variable's name from "param.delta" to "param"，because the parameter we send is "param.delta"，but recv "param".